ide/ide_slave.ml

```diff
@@ -374,8 +374,8 @@ let init =
    else begin
      let init_sid = Stm.get_current_state ~doc:(get_doc ()) in
      initialized := true;
-     let (ver,branch) = Envars.coq_version () in
-     Feedback.msg_info Pp.(str "You are running Coq " ++ str ver ++ str " (" ++ str branch ++ str ")");
+     let open Coqversion in
+     Feedback.msg_info Pp.(str "You are running Coq " ++ str version.describe ++ str " (" ++ str version.branch ++ str ")");
```

Comment:
When HEAD is detached, we get two sets of enclosing parentheses. Not a big deal though.

Reply:
In fact, it can be a big deal in some cases because of shell escaping (when a script calls `coqtop -v`). Maybe we should have a specific treatment of the detached head case.

Reply:
What should be done to fix that?

Reply:
One way would be to check if the outer characters are already parentheses :P.